src/ResourceManagement/DataFactory/DataFactory.Tests/Framework/JsonSamples/DatasetJsonSamples.cs

```diff
@@ -640,5 +640,53 @@ public class DatasetJsonSamples : JsonSampleCollection<DatasetJsonSamples>
     } 
 } 
 ";
+
+        [JsonSample]
+        public const string MongoDbDataset = @"
```

Comment:
Yes, take Dataset Json strings as example, they will be verified by test: **DatasetJsonConstsToWrappedObjectTest**(JsonSampleInfo sampleInfo).

Similarly, there are LinkedServiceJsonConstsToWrappedObjectTest, PipelineJsonConstsToWrappedObjectTest, and PipelineValidateJsonConstsTest, etc., to validate these json samples.